### ISSUE TITLE
Do not make openscap distcheck with -k

### ISSUE THREAD

**Issue by isimluk** [OpenSCAP/jenkins#9]
It is major PITA. I am swearing pretty bad. I do not like PITA.

In the dark past we have enabled `make distcheck -k` for nightly automation. That was during the times of irregular nightly builds. It made a kinda sense, because we wanted to see all the failures.

However, now things have changed. We have automation before&after each pull-request. We want to encourage people to look at the failed tests and see immediately what went wrong. Now, when uneducated user see the failed log from openscap jenkins, they basically have low chance to spot the failure. It takes me many many seconds before I get to the first failure. It makes me feel sick. It makes me not click on the jenkins result. I don't remember, if there were ever two failures in the log since the migration to public jenkins.

In conclusion the pain put on the shoulders of log readers is unjustifiable. I am very concerned. :-)

**Comment by isimluk** [comment]
Any objections to remove `-k` from the jenkins set-ups?

**Comment by jan-cerny** [comment]
I have no objection because this change will simplify my workflow.

**Comment by ybznek** [comment]
I would like to have maint with `distcheck -k`. Usually it is only for us. We can have some issue with some "I_sometimes_fail_probe" and we will don't know, if the fail didn't cover any other issue which wasn't  tested bellow.

I don't have any strong opinion to `-k` option for pull requests.

**Comment by isimluk** [comment]
Right. I don't have the strong opinion on maint tests, some of my arguments apply only on pull-requests job. :)

**Comment by ybznek** [comment]
No objections; I have deleted '-k'. (I have just now problem to find FAIL in tests, too :D )

**Comment by jan-cerny** [comment]
@ybznek Great! Thank you.

**Comment by isimluk** [comment]
Thank You!
